### ISSUE TITLE
[FEATURE REQUEST] Function to edit label names

### ISSUE THREAD

**Issue by BMSvsDave** [diffractor/diffractor#152]
@ZacWalk  It can be added the function to edit label names

**Comment by ZacWalk** [comment]
At the moment Diffractor supports label names as commony used in Adobe producs. You can change a files label fro the right click menu as below. Is this what you were asking?

![image](https://github.com/diffractor/diffractor/assets/181544/1dbc4ac4-df81-4605-98c6-98e1d2931cc7)

**Comment by BMSvsDave** [comment]
no, I really mean editing/changing the names of labels and ratings
![Immagine](https://github.com/diffractor/diffractor/assets/116594326/8415f34a-e641-41ff-a488-dc44048aefa5)

**Comment by BMSvsDave** [comment]
@ZacWalk  I wanted to manually change the name of the labels as I showed you in the photo above, is it possible to do in the next update?

Currently the names of the labels are

Reject
Select
Second
Approved
Review
To Do
